charts/consul/values.yaml

```diff
@@ -776,6 +776,10 @@ server:
   # @type: string
   logLevel: ""
 
+  # If false locality of server nodes won't be stored 
+  # @type: boolean
+  locality: true
```

Comment:
Should this be (in case we decide to further parameterize the init container's logic)

```
locality:
   enabled: true
```
instead?

Reply:
we can.

Reply:
https://github.com/hashicorp/consul-k8s/pull/2892/commits/f52bdd72824ba2f2f1488f14450dc95bccd227d2